e2e/newenemy/newenemy_test.go

```diff
@@ -276,11 +297,19 @@ func iterationsForHighConfidence(samples []int) (iterations int) {
 	return
 }
 
+var goodNs *NamespaceNames
+
 func checkDataNoNewEnemy(ctx context.Context, t testing.TB, slowNodeID int, crdb cockroach.Cluster, spicedb spice.Cluster, maxAttempts int, exitEarly bool) (bool, int) {
-	prefix := namespacesForNode(ctx, t, crdb[1].Conn(), spicedb[0].Client().V1Alpha1().Schema(), slowNodeID)
-	t.Log("filling with data to span multiple ranges for prefix", prefix)
 	objIDGenerator := generator.NewUniqueGenerator(objIDRegex)
-	fill(ctx, t, spicedb[0].Client().V1().Permissions(), prefix, objIDGenerator, 100, 100)
+	var prefix NamespaceNames
+	if goodNs == nil {
```

Comment:
if we get reversed timestamps in a namespace, it seems to much more likely to get subsequent reversed timestamps with the same namespaces, so we just keep using a good one until it doesn't work anymore